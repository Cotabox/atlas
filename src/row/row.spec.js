import { shallowMount } from '@vue/test-utils';
import Row from './row.vue';

describe('Row', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(Row);
  });

  it('should match with the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
