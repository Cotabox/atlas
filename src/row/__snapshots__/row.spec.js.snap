// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should match with the snapshot 1`] = `<div class="a-row is-justify-flex-start is-align-flex-start is-content-flex-start is-direction-row is-wrap" style="margin-left: -10px; margin-right: -10px;"></div>`;
